packages/connector-types/src/MediaConnector.ts

```diff
@@ -0,0 +1,23 @@
+import { Dictionary } from './Connector.Shared';
+
+export interface Media<MediaType> {
+    id: string;
+    name: string;
+    relativePath: string;
+    metaData: Dictionary;
+    extension?: string;
+    type: MediaType;
+}
+
+export type MediaDetail<MediaType> = Media<MediaType> & {
+    width?: number;
+    height?: number;
+};
+
+export type MediaConnectorCapabilities = {
+    query: boolean;
+    detail: boolean;
+    filtering: boolean;
+    metadata?: boolean;
+    upload?: boolean;
+};
```

Comment:
I know it didn't exist beforehand but what is the reasoning on why we don't expose `analyzeSmartCrop` capability?